src/apps/courierTracking.tsx

```diff
@@ -0,0 +1,196 @@
+import styled from "styled-components";
+import { isTriggered } from "../lib/utilityApis";
+import { InstantAppProps, queryToDataInput, queryToDataOutput } from "./types";
+
+const _ = require("lodash");
```

Comment:
```suggestion
import _ from 'lodash'
```

Mixed import/require statements 😄